lib/lambda_ethereum_consensus/fork_choice/fork_choice.ex

```diff
@@ -105,7 +105,6 @@ defmodule LambdaEthereumConsensus.ForkChoice do
 
   @impl GenServer
   def handle_call({:on_block, block_root, %SignedBeaconBlock{} = signed_block}, _from, state) do
-    Logger.info("[Fork choice] Adding block #{signed_block.message.slot} to the store.")
```

Comment:
This log is useful for checking the time spent on processing each block

Reply:
Should I add a "time passed" on the other log? This one:  `Logger.info("[Fork choice] Block #{slot} added to the store.")`

Reply:
Otherwise I feel we have to many logs per block.

Reply:
Let's just remove it. We can add a telemetry event later